index.html

```diff
@@ -618,6 +618,93 @@ <h4>
         </section>
       </section>
     </section>
+    <section id="automation" class="informative">
```

Comment:
If you say a section is informative, it is non-normative, which means it's not specifying anything (which is why the Examples section below was marked as informative).

Reply:
Fixed in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/bae1a77ba70c568c1f67c9b6c3470427c90c73ee